django/contrib/admin/options.py

```diff
@@ -829,6 +829,15 @@ def get_action(self, action):
             description = capfirst(action.replace('_', ' '))
         return func, action, description
 
+    def get_empty_value_display(self):
```

Comment:
Yes, wouldn't be any fun to have to set that on every field in all your models. :-)